app/root.css

```diff
@@ -79,7 +79,7 @@ input {
 
 .page {
   margin: auto;
-  max-width: 1608px; /* multiple of 12 for .col-x, 1 col at max width = 134px */
+  max-width: 1440px;
```

Comment:
I've made the grid (widths) static, so I think this line is just obsolete. But that's fine